charts/argo-workflows/templates/server/server-deployment-pdb.yaml

```diff
@@ -1,5 +1,5 @@
 {{- if and .Values.server.enabled .Values.server.pdb.enabled -}}
-apiVersion: policy/v1beta1
+apiVersion: {{ include "argo-workflows.podDisruptionBudget.apiVersion" . }}
```

Comment:
Specification changes between api versions are only below, so I think this change is harmless. :eyes:
https://kubernetes.io/docs/tasks/run-application/configure-pdb/#specifying-a-poddisruptionbudget

<img width="1017" alt="Screen Shot 2022-05-21 at 23 13 49" src="https://user-images.githubusercontent.com/30188755/169655596-ccf0290c-37ea-44a2-8e07-7bd88965064c.png">